src/main/scala/wdlTools/util/Util.scala

```diff
@@ -17,45 +17,20 @@ object Verbosity extends Enumeration {
 }
 import Verbosity._
 
-// a path to a file or an http location
-//
-// examples:
-//   http://google.com/A.txt
-//   https://google.com/A.txt
-//   file://A/B.txt
-//   foo.txt
-case class URL(addr: String)
-
-object URL {
-  def fromPath(path: Path): URL = {
-    fromUri(path.toUri)
-  }
-
-  def fromUri(uri: URI): URL = {
-    URL(uri.toString)
-  }
-}
-
 /**
   * Common configuration options.
   * @param localDirectories local directories to search for imports.
   * @param followImports whether to follow imports when parsing.
   * @param verbosity verbosity level.
   * @param antlr4Trace whether to turn on tracing in the ANTLR4 parser.
   */
-case class Options(localDirectories: Seq[Path] = Vector.empty,
+case class Options(localDirectories: Option[Iterable[Path]] = None,
```

Comment:
Can we use `Vector` instead of the more general `Iterable` ?

Reply:
yes I will fix this in the next PR